doc/api/v8.md

```diff
@@ -1,17 +1,29 @@
 # V8
 
-    Stability: 2 - Stable
+The `v8` module exposes APIs that are specific to the version of [V8][]
+built into the Node.js binary. It can be accessed using:
 
-This module exposes events and interfaces specific to the version of [V8][]
-built with Node.js.  These interfaces are subject to change by upstream and are
-therefore not covered under the stability index.
+```js
+const v8 = require('v8');
+```
+
+The APIs are subject to change at any time.
```

Comment:
Not just the APIs, their behavior as well.  The paragraph you replaced captures it better, IMO.